### PR TITLE
Embed last visited pages into user identity.

### PR DESCRIPTION
**Pull request by Hyperkid123**
Removes the user identity relation and embeds the last visited pages.

### CODE REVIEW

File: rest/routes/lastVisited.go [hunk 1]
```diff
@@ -25,18 +26,20 @@ func StoreLastVisitedPages(w http.ResponseWriter, r *http.Request) {
...
 	}
 
-	err = service.HandlePostLastVisitedPages(recentPages.Pages, userId)
+	err = service.HandlePostLastVisitedPages(recentPages.Pages, user)
 	if err != nil {
 		panic(err)
 	}
-	pages, err := service.GetUsersLastVisitedPages(userId)
+
+	fmt.Println("&&&&&&&&&&&&&&", user.LastVisitedPages.Data())
```
Comment (BlakeHolifield) [inline rest/routes/lastVisited.go:34]: You can keep it, just move it to a debug logger :)

**Review by BlakeHolifield — APPROVED** [review]
One small comment. Looks solid overall. We'll see how the migrations go in ephemeral

File: rest/routes/lastVisited.go (hunk 1 above)
Comment (Hyperkid123) [reply rest/routes/lastVisited.go:34]: :rofl: no I was just trying to figure out what the .Data does

**Pull request merged**